docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1576,6 +1578,28 @@ The log level of individual log names can be overridden. For example, to mute al
 </logger>
 ```
 
+**Regular Expression Filtering**
+
+The messages logged can be filtered using regular expressions using `message_regexp` and `message_regexp_negative`. This can be done on a per-level basis or globally. If both a global and logger-specific pattern is specified, the global pattern is overridden (ignored) and only the logger-specific pattern applies. The positive and negative patterns are considered independently for this situation. Note: Using this feature may cause a slight slowdown in performance.
+
+
+```xml
+    <logger>
+        <level>trace</level>
+        <!-- Global: Don't log and Trace messages -->
```

Comment:
```suggestion
        <!-- Global: Don't log Trace messages -->
```

Reply:
Actually, I just re-read this. The original version was correct. I just updated it to the following to be completely clear. `Global: Don't log messages that have "Trace" in them`